src/v2/options.ts

```diff
@@ -217,8 +224,22 @@ export function getGlobalOptions(): GlobalOptions {
  * Additional fields that can be set on any event-handling Cloud Function.
  */
 export interface EventHandlerOptions extends GlobalOptions {
+  eventType?: string;
+
+  eventFilters?: Record<string, string | Expression<string>>;
+  eventFilterPathPatterns?: Record<string, string | Expression<string>>;
```

Comment:
what's going on here? We don't expect users to set these options no?

Reply:
...I wasn't aware of that, actually, but having actually looked at the relevant code paths, i think you're right.

Reply:
I think we may actually need to support expressions here. For example, a bucket or RTDB collection could be an expression.